app/mailers/admin_mailer.rb

```diff
@@ -24,4 +24,24 @@ def submission(submission:, user:, artist:)
       subject: "Submission ##{@submission.id}"
     ) { |format| format.html { render layout: "mailer_no_footer" } }
   end
+
+  def artwork_updated(submission:, user_data:, artwork_data:, changes:)
+    assigned_admin = AdminUser.find_by(gravity_user_id: submission.assigned_to)
+
+    @submission = submission
+    @user_id = submission.user.gravity_user_id
+    @user_email = submission.user.email
+    @artwork_id = artwork_data[:id]
+    @changes = changes
+
+    smtpapi category: %w[submission],
+            unique_args: {
+              submission_id: submission.id
+            }
+
+    mail(
+      to: 'nikita.skalkin@artsymail.com', # assigned_admin.email,
```

Comment:
going to switch it after QA on staging & letting secondary team know about this change